README.md

```diff
@@ -1,66 +1,38 @@
 # freezeyt
-Generator of static web pages created by the Czech community of PyLadies.
-
-
-## Table of contents
-* [Project goal](#Project-goal)
-* [What is the project about?](#What-is-the-project-about?)
-    * [What is NOT the project about?](#What-is-NOT-the-project-about?)
-* [Installation](#Installation)
-* [Usage](#Usage)
-* [Contributing](#Contributing)
-    * [How to start contribute](#How-to-contribute)
-* [Authors](#Authors)
-* [Licence](#Licence)
-    
+Generator of static web pages created by the Czech community of python programming language.
```

Comment:
Možná změnit na:
`... by the Czech Python community.`
minimálně Python, prosím, velkým.

Reply:
v těch velkých písmenech mám už chaos :D, česky bych tam napsal malé ne ? Plzeň x plzeňské. Jak je to v angličtině ?

Reply:
V angličtině velkými písmeny, např. Czech. Python, Slack, Flask, Pythonistas bych také psala s velkým počátečním písmenem. Je to tam na více místech.